ui/page/creatorMemberships/internal/overviewTab/style.scss

```diff
@@ -0,0 +1,98 @@
+.bank-account-status {
+  margin-top: var(--spacing-s);
+  padding: var(--spacing-s);
+  border-radius: var(--border-radius);
+  border: 2px solid rgba(255, 0, 0, 0.8);
+  background-color: rgba(255, 0, 0, 0.1);
+
+  div {
+    label {
+      display: block;
+      font-size: var(--font-body);
+      font-weight: var(--font-weight-bold);
+    }
+
+    span {
+      display: block;
+      font-size: var(--font-small);
+      font-weight: var(--font-weight-bold);
+      opacity: 0.8;
+    }
+  }
+
+  .button {
+    width: auto;
+    margin-top: var(--spacing-s);
+    margin-bottom: 0;
+  }
+}
+
+.membership__mychannels-header {
+  position: relative;
+  width: 100%;
+  background-size: cover;
+  background-position: top -320px left 0px;
+  background-image: url('/public/img/my_channels.jpg');
```

Comment:
This doesn't work with well with localhost.  

<img width="518" alt="image" src="https://user-images.githubusercontent.com/64950861/190136627-739f2114-1461-46ad-bac4-7949ed983ed3.png">

The following should be the standard way:
- `background-image: url('~static/img/my_channels.jpg')`;
- Remove `to: '${output.PATH}/public/img/my_channels.jpg'` since the transfer would then be automatic.

----

Having said that, these images should be from CDN, so I guess this is just temporary.  
Submitted it just to keep track in case we forget.